### PR TITLE
2649: Build beta global nav

### LINKED ISSUE #2649: Global nav build

### PR DESCRIPTION
**Pull request by Caleb-Ellis**
## Done

- Built global nav according to [copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y) and [design](https://github.com/canonical-websites/www.ubuntu.com/issues/2643)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check that the global nav matches the [copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y) and [design](https://github.com/canonical-websites/www.ubuntu.com/issues/2643)
- Check that it closes if you click outside of it (if it's open)

## Issues

Fixes #2649

### CODE REVIEW

**Comment by frankiegranato** [comment]
Looks good to me. @pmahnke may want to review this too.

**Comment by frankiegranato** [comment]
Sorry @Caleb-Ellis on further consideration, I've had an afterthought. Can we add a 'Contact' link to the global nav? I've made an amend in the [copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#) to this effect. Please don't bother adding it if @pmahnke thinks its unnecessary or if its a lot of extra bother. As I say, it was just an afterthought :-)

**Comment by kwm14** [comment]
## Design review
Looking good @Caleb-Ellis 👍 just one small update
- [ ] reduce height of Canonical logo to `10px` making sure both Canonical and Ubuntu logos line up

**Comment by Caleb-Ellis** [comment]
@kwm14 done
@frankiegranato I've added a 'Contact' link and it should show on the demo. No bother at all! :)

**Comment by kwm14** [comment]
Top man. Design 👍

File: templates/templates/global-nav.html [hunk 1]
```diff
@@ -33,63 +33,63 @@
...
-            <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}0de4fcd5-logo-maas-icon.svg" alt="icon">
+            <a href="https://maas.io/"><img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}0de4fcd5-logo-maas-icon.svg" alt="icon"></a>
             <div class="p-matrix__content">
               <h4 class="p-matrix__title"><a class="p-link" href="https://maas.io/">MAAS&nbsp;&rsaquo;</a></h4>
               <p class="p-matrix__desc">Create a bare-metal cloud with Metal as a Service for IPAM and provisioning</p>
             </div>
           </li>
           <li class="p-matrix__item">
-            <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}4964d639-landscape-logo.svg" alt="icon">
+            <a href="http://landscape.canonical.com/"><img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}4964d639-landscape-logo.svg" alt="icon"></a>
```
Comment (pmahnke) [inline templates/templates/global-nav.html]: https please - https://landscape.canonical.com/

File: templates/templates/global-nav.html [hunk 2]
```diff
@@ -33,63 +33,63 @@
...
-            <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}a7916513-picto-openstack.svg" alt="icon">
+            <a href="https://www.ubuntu.com/openstack"><img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}a7916513-picto-openstack.svg" alt="icon"></a>
             <div class="p-matrix__content">
               <h4 class="p-matrix__title"><a class="p-link" href="https://www.ubuntu.com/openstack">OpenStack&nbsp;&rsaquo;</a></h4>
               <p class="p-matrix__desc">The world’s first choice for OpenStack - the leader in density and cost per VM</p>
             </div>
           </li>
           <li class="p-matrix__item">
-            <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}b81a45e4-kubernetes.svg" alt="icon">
+            <a href="http://www.ubuntu.com/kubernetes"><img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}b81a45e4-kubernetes.svg" alt="icon"></a>
```
Comment (pmahnke) [inline templates/templates/global-nav.html]: https please

File: templates/templates/global-nav.html [hunk 3]
```diff
@@ -147,6 +147,7 @@ <h5 class="p-muted-heading"><strong>About</strong></h5>
             <li class="p-list__item"><a href="https://insights.ubuntu.com/press-centre">Press centre</a></li>
             <li class="p-list__item"><a href="https://partners.ubuntu.com">Partners</a></li>
             <li class="p-list__item"><a href="https://buy.ubuntu.com">Merchandise</a></li>
```
Comment (pmahnke) [inline templates/templates/global-nav.html]: merch is at https://shop.canonical.com/

**Comment by Caleb-Ellis** [comment]
@pmahnke I've fixed the links, but the GitHub code viewer doesn't seem to reflect it, weirdly. If you pull the code or just go to the demo you'll see the right links. Maybe a caching issue with GitHub?

**Pull request merged**